app/models/invitation_message_builder.rb

```diff
@@ -34,6 +34,12 @@ def success_message
     end
   end
 
+  def check_for_emails(emails)
+    if emails.empty?
+      add_error('Must enter a valid email.') 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.